kafka-connect/build.gradle

```diff
@@ -64,9 +64,14 @@ project(':iceberg-kafka-connect:iceberg-kafka-connect-runtime') {
   configurations {
     hive {
       extendsFrom runtimeClasspath
+      // force upgrades for dependencies with known vulnerabilities...
+      resolutionStrategy {
+        force 'commons-beanutils:commons-beanutils:1.11.0'
+      }
     }
     all {
       exclude group: 'javax.activation', module: 'activation'
+      exclude group: 'commons-beanutils'
```

Comment:
The checkstyle fails because of this line. Do we really need to exclude it? If so, can we try giving a module?

Reply:
My goal here is to not include it when not building for hive. But I'll file a separate issue for that and work on that for a future release. We don't need these hadoop deps for the non-Hive build. I'll update my PR shortly.

Reply:
Yes. We should focus on fixing CVE for main distribution (not hive) first. As that is the plugin we are gonna upload to confluent repo.